esp/public/media/scripts/onsite/teacher_checkin.js

```diff
@@ -108,28 +118,54 @@ $j(function(){
         this.value += "...";
     });
 
+    // Undo the check in of the user with the specified username, assuming
+    // that that user was not checked in when this page was loaded (because
+    // the DOM would look different depending on whether that were true). If
+    // no username specified, undo the last check in.
+    function undoLiveCheckIn(username) {
+        var targetCheckin;
+        if (username === undefined) {
+            if (checkins.length === 0) return false;
```

Comment:
Here and elsewhere, let's do a separate line (and braces) for the body to be explicit, it's just easier to read and harder to mess up that way.